### ISSUE TITLE
Release Candidate Comments (Documentation Accessibility)

### ISSUE THREAD

**Issue by GeorgLink** [chaoss/wg-diversity-inclusion#321]
This issue was created to collect comments about the next metrics release.

This thread is for comments about the Documentation Accessibility metric.

Release candidate at: https://chaoss.community/metric-documentation-accessibility/
GitHub location: https://github.com/chaoss/wg-diversity-inclusion/blob/master/focus-areas/project-and-community/documentation-accessibility.md

**Comment by LawrenceHecht** [comment]
#339 is related to this. I don't think document accessibility should be a separate metric. "Information Accessibility" is a better way to think of things. In a broader way of thinking of things, it can be Communication Accessibility, which @geekygirldawn typed up 2 years ago: https://github.com/chaoss/wg-diversity-inclusion/tree/master/focus-areas/communication

**Comment by rpaik** [comment]
The matrix rows for surveying community members seem focused on topics related to governance or community. What about gauging how easy it is to find information on the software, features, technical project, etc?

**Comment by germonprez** [comment]
Thanks @rpaik How about including these: 

- Installation procedures
- Feature development 
- API integration
- Test suites

Thinking off the top of my head.

**Comment by rpaik** [comment]
> Thanks @rpaik How about including these:
> 
> * Installation procedures
> * Feature development
> * API integration
> * Test suites
> 
> Thinking off the top of my head.

Thanks @germonprez. How about also adding: 
* Architecture overview
* User guide

**Comment by germonprez** [comment]
I'll make a PR for this.

**Comment by rpaik** [comment]
Thanks @germonprez!

**Comment by germonprez** [comment]
Done! Pinged you as a reviewer @rpaik :)

**Comment by klumb** [comment]
Closing for release
